templates/components/_modal.html

```diff
@@ -1,43 +1,58 @@
 <!-- Modal Structure -->
-<div class="modal fade" id="job_apply" tabindex="-1" aria-labelledby="Job-Application-Modal" aria-hidden="true">
-    <div class="modal-dialog">
-        <div class="modal-content">
-            <div class="modal-header">
-                <h3 class="modal-title">Application for <span class="text-red">{{ job.role }}</span> with 
-                    <span class="text-red">{{ job.employer }}</span>
-                </h3>
-                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
-            </div>
-            <div class="modal-body">
-                
-                    Applicant: <span class="ps-3">{{ user.full_name }}</span> <br>
-                    Email: <span class="ps-3">{{ user.email }}</span> <br>
+<form class="col" method="POST" action="{{ url_for('job_applied', job_id=job._id) }}">
+    <div class="modal fade" id="job_apply" tabindex="-1" aria-labelledby="Job-Application-Modal" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h3 class="modal-title">Application for <span style="color: var(--s1-color);">{{ job.role }}</span> with 
+                        <span style="color: var(--s1-color);">{{ job.employer }}</span>
```

Comment:
Inline CSS use....can this be changed and identified in the external custom css file to maintain Separation of Concern Code principles?